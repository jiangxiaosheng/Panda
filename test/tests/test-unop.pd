func main() {
	var a: boolean = false
    a
    !a
}