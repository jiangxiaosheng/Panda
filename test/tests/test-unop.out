Sementically checked program: 


void main() {
var a: bool(bool : false)
(bool : a)
(bool : !(bool : a))
}